easybuild/easyblocks/i/imkl.py

```diff
@@ -218,9 +246,8 @@ def post_install_step(self):
             # blas95 and lapack also need include/.mod to be processed
             fftw2libs = ['fftw2xc', 'fftw2xf']
             fftw3libs = ['fftw3xc', 'fftw3xf']
-            cdftlibs = ['fftw2x_cdft']
-            if LooseVersion(self.version) >= LooseVersion('10.3'):
-                cdftlibs.append('fftw3x_cdft')
+            # cdftlibs set in prepare_step
+            cdftlibs = self.cdftlibs
```

Comment:
@bartoldeman There's no point in doing this, should use `self.cdftlibs` everywhere and just stop using `cdftlibs`?